docs/installation.md

```diff
@@ -26,101 +31,92 @@ mv nextflow ~/bin
 
 ```bash
-git clone https://github.com/nf-core/methylseq.git
-nextflow run nf-core/methylseq/bismark.nf
-nextflow run nf-core/methylseq/bwa-meth.nf # Alternative bwa-meth pipeline
+# Download the latest release of the pipeline (see https://github.com/nf-core/methylseq/releases)
+curl -L https://github.com/nf-core/methylseq/archive/1.0.zip -o nf-core-methylseq-v1.0.zip
+unzip nf-core-methylseq-v1.0.zip
+cd /path/to/my/data
+nextflow run /path/to/pipelines/nf-core-methylseq-v1.0 [parameters]
 ```
 
-## 3) Pipeline Configuration
+#### 2.3) Development
+
+If you would like to make changes to the pipeline, it's best to make a fork on GitHub and then clone the files. Once cloned you can run the pipeline directly as above.
+
+
+## 3) Pipeline configuration
 By default, the pipeline runs with the `standard` configuration profile. This uses a number of sensible defaults for process requirements and is suitable for running on a simple (if powerful!) basic server. You can see this configuration in [`conf/base.config`](../conf/base.config).
 
 Be warned of two important points about this default configuration:
 
 1. The default profile uses the `local` executor
     * All jobs are run in the login session. If you're using a simple server, this may be fine. If you're using a compute cluster, this is bad as all jobs will run on the head node.
+    * See the [nextflow docs](https://www.nextflow.io/docs/latest/executor.html) for information about running with other hardware backends. Most job scheduler systems are natively supported.
 2. Nextflow will expect all software to be installed and available on the `PATH`
 
-The default resource requests are all measured against the following default limits:
-
-* `max_memory` = 128.GB
-* `max_cpus` = 16
-* `max_time` = 240.h
-
-To adjust these limits, specify them on the command line with two hyphens, eg. `--max_memory '64.GB'`.
+#### 3.1) Software deps: Docker and Singularity
+Running the pipeline with the option `-with-singularity` or `-with-docker` tells Nextflow to enable either [Singularity](http://singularity.lbl.gov/) or Docker for this run. An image containing all of the software requirements will be automatically fetched and used (https://hub.docker.com/r/nf-core/methylseq).
 
-Note that these limits are the maximum to be used _per task_. Nextflow will automatically attempt to parallelise as many jobs as possible given the available resources.
+If running offline with Singularity, you'll need to download and transfer the Singularity image first:
 
-## 3.2) Configuration: Docker and Singularity
-Running the pipeline with the option `-with-docker` tells Nextflow to enable docker for this run. The above base profile already specifies the container to be used (hub.docker.com/r/nf-core/methylseq).
-
-If you don't have Docker available and do have [Singularity](http://singularity.lbl.gov/), instead use `-with-singularity` on the command line.
+```bash
+# Switch 1.4 with the version of the pipeline that you're using
```

Comment:
I would directy use something like [VERSION] that way, it's pretty clear that you have to change version, and the doc is always up to date ;-)